### PR TITLE
Add cols_view to relevant ops

### PR DESCRIPTION
**Pull request by Phlya**
Addresses https://github.com/open2c/bioframe/issues/101

I simply rename columns in the view internally to match the df, trying to preserve them throughout was a bit troublesome.

### CODE REVIEW

File: bioframe/ops.py [hunk 1]
```diff
@@ -1382,14 +1388,15 @@ def trim(
...
+    ckv, skv, ekv = _get_default_colnames() if cols_view is None else cols_view
     view_df = construction.make_viewframe(
-        view_df, view_name_col=view_name_col, cols=cols
-    )
+        view_df, view_name_col=view_name_col, cols=[ckv, skv, ekv]
+    ).rename(columns=dict(zip([ckv, skv, ekv], [ck, sk, ek])))
 
     if inferred_view:
-        print('correct')
+        print("correct")
```
Comment (gfudenberg) [inline bioframe/ops.py:1397]: can we delete this print?

Comment (gfudenberg) [reply bioframe/ops.py:1397]: also can line 1423 be moved earlier, with the other inferred view stuff?

```python
if inferred_view:
        view_name_col = ck```

Comment (gfudenberg) [reply bioframe/ops.py:1397]: otherwise looks good-- thanks @Phlya !

Comment (Phlya) [reply bioframe/ops.py]: Thank you @gfudenberg !

**Comment by Phlya** [comment]
OK, done, shall we merge?

**Pull request merged**